### PR TITLE
Move server cleaner into YAML job

### PR DESCRIPTION
**Pull request by Sanchiz**
This is the first step for moving business login from Jenkins jobs into playbooks. It will allow us to use Code Driven Development for Jenkins infrastructure.

### CODE REVIEW

File: core/cibox-jenkins/templates/jobs/server_cleaner.xml.j2 [hunk 1]
```diff
@@ -17,14 +52,11 @@
   <concurrentBuild>false</concurrentBuild>
   <builders>
     <hudson.tasks.Shell>
-      <command>
-mysql -uroot -proot -e &quot;show databases;&quot; -s | egrep &quot;^drupal&quot; | xargs -I &quot;@@&quot; mysql -uroot -proot -e &quot;DROP DATABASE @@&quot;
-
-cd /var/www/
-sudo rm -rf build*
+      <command>export PYTHONUNBUFFERED=1
+ansible-playbook sourcecode/cibox/jobs/server_cleaner.yml -i &apos;localhost,&apos; --connection=local
```
Comment (Sanchiz) [inline core/cibox-jenkins/templates/jobs/server_cleaner.xml.j2]: Here we run only playbook. That should be the rule for all jobs.

File: core/cibox-project-builder/files/vagrant/box/cibox/jobs/server_cleaner.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  connection: local
+  gather_facts: yes
+
+  tasks:
+  - name: Clean database
+    shell: 'mysql -uroot -proot -e "show databases;" -s | egrep "^drupal" | xargs -I "@@" mysql -uroot -proot -e "DROP DATABASE @@"'
+    sudo: yes
+
+  - name: Remove builds
+    shell: 'rm -rf /var/www/build*'
```
Comment (podarok) [inline core/cibox-project-builder/files/vagrant/box/cibox/jobs/server_cleaner.yml]: it is not full
Let's incorporate YMCA cleaner commands here

File: core/cibox-jenkins/templates/jobs/server_cleaner.xml.j2 (hunk 1 above)
Comment (podarok) [inline core/cibox-jenkins/templates/jobs/server_cleaner.xml.j2]: @Sanchiz how the cleaner is set up into sourcecode folder?

**Comment by Sanchiz** [comment]
Todo:
```
cd /tmp
sudo find . -maxdepth 1 -name '*.pdf' -delete
sudo find . -maxdepth 1 -name '*.jpg' -delete
sudo find . -maxdepth 1 -name '*.mp4' -delete
sudo find . -maxdepth 1 -name '*.avi' -delete
sudo find . -maxdepth 1 -name '*.png' -delete
sudo find . -maxdepth 1 -name '*.csv' -delete
sudo find . -maxdepth 1 -name '*.mov' -delete
sudo find . -maxdepth 1 -name '*.doc' -delete
sudo find . -maxdepth 1 -name '*.dotx' -delete
sudo find . -maxdepth 1 -name '*.docx' -delete
sudo find . -maxdepth 1 -name '*.xlsx' -delete
sudo find . -maxdepth 1 -name '*.xls' -delete
sudo find . -maxdepth 1 -name '*.swf' -delete
sudo find . -maxdepth 1 -name '*.wmv' -delete
sudo find . -maxdepth 1 -name 'file*' -delete
sudo find . -maxdepth 1 -name '*.gif' -delete
sudo find . -maxdepth 1 -name '*.mp3' -delete
sudo rm -rf /tmp/confi*
sudo apt-get update || true
sudo rm -rf /var/log/jenkins/*
```

File: core/cibox-jenkins/templates/jobs/server_cleaner.xml.j2 (hunk 1 above)
Comment (Sanchiz) [inline core/cibox-jenkins/templates/jobs/server_cleaner.xml.j2]: Git clone from master branch.
image

Comment (podarok) [inline core/cibox-jenkins/templates/jobs/server_cleaner.xml.j2]: ok, agreed

**Pull request merged**